ansible/deployandtest.sh

```diff
@@ -138,9 +138,14 @@ ANSIBLE_VARS="$ANSIBLE_VARS cloud=$CLOUD"
 ANSIBLE_VARS="$ANSIBLE_VARS bootdelay=$BOOTDELAY"
 ANSIBLE_VARS="$ANSIBLE_VARS ansible_root=$cwd"
 ANSIBLE_VARS="$ANSIBLE_VARS uniqifier=$UNIQIFIER"
-ANSIBLE_VARS="$ANSIBLE_VARS ADMIN_PASSWORD=$ADMIN_PASSWORD"
+ANSIBLE_VARS="$ANSIBLE_VARS admin_password=$ADMIN_PASSWORD"
```

Comment:
Because if I am using Vault I wouldn't expect to have to set this variable, right?

Reply:
Correct. If you don't set it, then a default will be set.
If you set the vault setting, then it is not used.
We could add a warning that you have set an admin AND set a vault password?

Reply:
I think that warning will one day save me from confusion, so let's do that. Thanks!

Reply:
Add a full error to ensure the mistake is caught